$NetBSD: patch-aa,v 1.7 2000/08/22 21:23:24 skrll Exp $

The libtool staff pulled to KDE in rev. 331 of
	kde-common/admin/acinclude.m4.in (jdolecek)

--- configure.orig	Tue Sep  7 16:24:53 1999
+++ configure	Mon Aug 14 17:50:44 2000
@@ -5479,7 +5479,7 @@
 
 
 
-
+if false; then
 echo $ac_n "checking for main in -ljs""... $ac_c" 1>&6
 echo "configure:5485: checking for main in -ljs" >&5
 ac_lib_var=`echo js'_'main | sed 'y%./+-%__p_%'`
@@ -5525,7 +5525,7 @@
 For this have a look at http://people.ssh.fi/mtr/js
 " 1>&2
  	TOPSUBDIRS="$TOPSUBDIRS js"
-	
+fi	
 
 cat >> confdefs.h <<EOF
 #define CANONICAL_HOST "$host"
